packages/cli-kit/src/public/node/node-package-manager.ts

```diff
@@ -29,6 +29,13 @@ export const pnpmWorkspaceFile = 'pnpm-workspace.yaml'
 
 /** An array containing the lockfiles from all the package managers */
 export const lockfiles: Lockfile[] = [yarnLockfile, pnpmLockfile, npmLockfile, bunLockfile]
+export const lockfilesByManager: {[key in PackageManager]: Lockfile} = {
+  yarn: yarnLockfile,
+  npm: npmLockfile,
+  pnpm: pnpmLockfile,
+  bun: bunLockfile,
+  unknown: yarnLockfile,
```

Comment:
it's weird that unknown is `yarnLockFile`, it should probably be just `undefined`? and make this type 
`{[key in PackageManager]: Lockfile | undefined}`

Reply:
Yeah, undefined probably makes sense here :)